### ISSUE TITLE
Looking for a method to get my ongoing games

### ISSUE THREAD

**Issue by KingKevin23** [tors42/chariot#3]
Hello again,

as the title says I am looking for a method to get my ongoing games. The Lichess API has the endpoint `https://lichess.org/api/account/playing` which gives you a list of your current games and I am looking for exactly that in the Java Client. I have found the method `client.games().ongoingGames()`, but this seems not to work. 

```java
ClientAuth client = Client.auth(token);
client.games().ongoingGames().stream().forEach(System.out::println);
```

The code above outputs just `NowPlaying[fullId=null, gameId=null, fen=null, color=null, lastMove=null, speed=null, perf=null, rated=false, isMyTurn=false, variant=null, opponent=null]` every single time, wether I am playing a game or not. If this is indeed the method I am looking for, I would appreciate a fix. Else I would be very grateful for a hint to the right direction!

Thanks in advance.

**Comment by tors42** [comment]
Ah, fixed in 0.0.38 - thanks for reporting!

**Comment by KingKevin23** [comment]
(Although it is wrapped but it seems you already figured that out ;-))
